polyfill/lib/absolute.mjs

```diff
@@ -103,16 +103,53 @@ export class Absolute {
   difference(other, options) {
     if (!ES.IsTemporalAbsolute(this)) throw new TypeError('invalid receiver');
     if (!ES.IsTemporalAbsolute(other)) throw new TypeError('invalid Absolute object');
-    const largestUnit = ES.ToLargestTemporalUnit(options, 'seconds', ['years', 'months', 'weeks', 'days']);
+    const disallowedUnits = ['years', 'months', 'weeks', 'days'];
+    const smallestUnit = ES.ToSmallestTemporalDurationUnit(options, 'nanoseconds', disallowedUnits);
+    let defaultLargestUnit = 'seconds';
```

Comment:
```suggestion
    const defaultLargestUnit = 'seconds';
```

Reply:
It's reassigned in the next line if `smallestUnit` is larger than `defaultLargestUnit`.